### PR TITLE
rm: boards: add flashing steps for uz3eg-iocc board

### PR DESCRIPTION
**Pull request by igoropaniuk**
Add flashing steps for uz3eg-iocc board.

Signed-off-by: Igor Opaniuk <igor.opaniuk@foundries.io>

## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for e21b184238aaf8c334212b87d5edbfad8cc416f8 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1697/docs/artifacts/html/index.html

File: source/reference-manual/boards/uz3eg-iocc.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Avnet UltraZed SOM with UltraZed-EG IO Carrier Card
+===================================================
+
+.. include:: generic-prepare.rst
+
+Hardware Preparation
+--------------------
+
+#. Ensure that the power switch is off (``SW8`` switch is off).
```
Comment (MrCry0) [inline source/reference-manual/boards/uz3eg-iocc.rst:11]: Do we have a board to include photos of switch SW8 and boot switches on SoM?

**Review by kprosise — APPROVED** [review]
LGTM

**Review by MrCry0 — APPROVED** [review]

**Comment by kprosise** [comment]
@igoropaniuk, I can go ahead and merge this if it is ready.

File: source/reference-manual/boards/uz3eg-iocc.rst (hunk 1 above)
Comment (igoropaniuk) [reply source/reference-manual/boards/uz3eg-iocc.rst:11]: Tbh all listed switches are pretty easy to find on the board and hard to confuse with something else. Anyway, we can add that in future, if needed

**Pull request merged**